app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
```

Comment:
Code Style Issue: The parameter names `name` and `age` are descriptive and accurately represent the data they hold. This is in line with the checklist item on using descriptive and correct variable names.